membership-common/src/main/scala/com/gu/salesforce/Scalaforce.scala

```diff
@@ -127,60 +125,69 @@ abstract class Scalaforce(implicit ec: ExecutionContext) extends LazyLogging {
 
   object Query {
     def execute(query: String): Future[\/[String, JsValue]] = {
-      val path = s"services/data/v29.0/query?q=$query"
-      Timing.record(metrics, "Execute query") {
-        get(path)
-      }.map { response =>
-        val bodyString = response.body().string() // out here to make sure connection is closed in all cases
-        response.code() match {
-          case Status.OK => \/-(jsonParse(bodyString))
-          case code => -\/(s"SF004: Salesforce returned code $code for query: $query")
+      val path = s"services/data/v57.0/query?q=$query"
+      Timing
+        .record(metrics, "Execute query") {
+          get(path)
+        }
+        .map { response =>
+          val bodyString = response.body().string() // out here to make sure connection is closed in all cases
+          response.code() match {
+            case Status.OK => \/-(jsonParse(bodyString))
+            case code => -\/(s"SF004: Salesforce returned code $code for query: $query")
+          }
         }
-      }
     }
   }
 
   object Contact {
     def read(key: String, id: String): Future[\/[String, Option[JsValue]]] = {
-      val path = s"services/data/v29.0/sobjects/Contact/$key/$id"
-      Timing.record(metrics, "Read Contact") {
-        get(path)
-      }.map { response =>
-        val bodyString = response.body().string() // out here to make sure connection is closed in all cases
-        response.code() match {
-          case Status.OK => \/-(Some(jsonParse(bodyString)))
-          case Status.NOT_FOUND => \/-(None)
-          case code => -\/(s"SF003: Salesforce returned code $code for Contact read $key $id")
+      val path = s"services/data/v57.0/sobjects/Contact/$key/$id"
```

Comment:
@graham228221  says: "This returns all available Contact fields, so moving from v29.0 to v57.0 will mean some additional fields are returned by Salesforce (but no fields will be removed, as far as I can tell)."